### DISCUSSION TITLE
Question about AI's interaction with the Sphinx

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4681] [Q&A] [answered]
Hello @ihhub,

I am curious about this: :)

"- Fix many issues with AI behavior on the map and add AI interaction with Alchemist Tower and **Sphinx**" (https://github.com/ihhub/fheroes2/releases)

What kind of interaction is effective in the game for the AI with the Sphinx?
Is the AI smart enough to answer a question like this? :rofl: 

"I walk on four legs at the beginning of my life, then on two legs in the middle of my life, and on three legs at the end of my life.

Who am I?"

**Comment by ihhub** [comment, marked as answer]
The game was crashing for Windows due to an assertion when an AI hero walks over Sphinx tile. We fixed it. AI still ignore Sphinx... for now.

**Comment by zenseii** [comment]
> "I walk on four legs at the beginning of my life, then on two legs in the middle of my life, and on three legs at the end of my life.
> 
> Who am I?"

In the future you will be able to ask the AI stuff like what the meaning of life is. 😜

**Comment by LeHerosInconnu** [comment]
This has been answered.
I close the discussion.
